### PR TITLE
[risk=no] Db 149 Hiding the synonyms 

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
1. Displaying only part of the synonyms unless user wants to view all.

### CODE REVIEW

File: public-ui/src/app/data-browser/chart/chart.component.ts [hunk 1]
```diff
@@ -101,7 +101,12 @@ export class ChartComponent implements OnChanges {
             style: this.isGenderIdentityAnalysis()
                 ? this.dbc.GI_DATA_LABEL_STYLE : this.dbc.DATA_LABEL_STYLE,
             distance: this.isGenderIdentityAnalysis() ? 3 : -30,
-            format: '{point.name} {point.percentage:.0f}%',
+            formatter: function () {
```
Comment (calbach) [inline public-ui/src/app/data-browser/chart/chart.component.ts]: nit: arrow function is preferred

File: public-ui/src/app/views/ehr-view/ehr-view.component.ts [hunk 2]
```diff
@@ -195,5 +190,11 @@ export class EhrViewComponent implements OnInit, OnDestroy {
       this.showGenderIdentity = false;
     }
   }
-
+  public toggleSynonyms(conceptId) {
+    if (!this.showMoreSynonyms[conceptId]) {
```
Comment (calbach) [inline public-ui/src/app/views/ehr-view/ehr-view.component.ts]: nit: would probably just do a `this.showMore[..] = !this.showMore[..]`

File: public-ui/src/app/views/ehr-view/ehr-view.component.ts [hunk 3]
```diff
@@ -38,6 +38,7 @@ export class EhrViewComponent implements OnInit, OnDestroy {
   private searchRequest: SearchConceptsRequest;
   private subscriptions: ISubscription[] = [];
   private initSearchSubscription: ISubscription = null;
+  showMoreSynonyms = {};
```
Comment (calbach) [inline public-ui/src/app/views/ehr-view/ehr-view.component.ts]: nit: would add a comment here describing what this property does

File: public-ui/src/app/views/ehr-view/ehr-view.component.ts (hunk 2 above)
Comment (SrushtiGangireddy) [reply public-ui/src/app/views/ehr-view/ehr-view.component.ts]: Done.

File: public-ui/src/app/views/ehr-view/ehr-view.component.ts (hunk 3 above)
Comment (SrushtiGangireddy) [reply public-ui/src/app/views/ehr-view/ehr-view.component.ts]: Done thanks.

File: public-ui/src/app/data-browser/chart/chart.component.ts (hunk 1 above)
Comment (SrushtiGangireddy) [reply public-ui/src/app/data-browser/chart/chart.component.ts]: This unfortunately seems to be tough as the arrow function gets the whole context of the document and unlike chart's context in this function. So, leaving it as it is for now.

Comment (calbach) [reply public-ui/src/app/data-browser/chart/chart.component.ts]: Ah I see - thanks for checking.

**Pull request merged**